beacon-chain/casper/validator_test.go

```diff
@@ -139,3 +139,55 @@ func TestValidatorIndices(t *testing.T) {
 		t.Errorf("exited validator indices should be [3], got: %v", ExitedValidatorIndices(data.Validators, data.CurrentDynasty))
 	}
 }
+
+func TestAreAttesterBitfieldsValid(t *testing.T) {
+	attestation := &pb.AttestationRecord{
+		AttesterBitfield: []byte{'F'},
+	}
+
+	indices := []uint32{0, 1, 2, 3, 4, 5, 6, 7}
+
+	isValid := AreAttesterBitfieldsValid(attestation, indices)
+	if !isValid {
+		t.Fatalf("expected validatoin to pass for bitfield %v and indices %v", attestation, indices)
```

Comment:
I feel `Errorf` is more appropriate here. Any reason why we have to use `Fatalf?

Reply:
Same with the tests below

Reply:
Hmm is there a difference? Either way this is the end of the test. 

@prestonvanloon and I discussed `Error` vs `Fatal` earlier. He argued for using `Fatal` and `Error` depending on the situation. I argued that it didn't really matter and `Fatal` was a good default 99% of the time.

Reply:
I see. Yeah, I agree there's no difference here.